demos/styles.yaml

```diff
@@ -9,6 +9,18 @@ camera:
 
     # type: flat
 
+sources:
```

Comment:
Can we not put this block in between camera and lighting? Those kinda go together. It could be before or after both.